### PR TITLE
Add froger OOP task

### PR DESCRIPTION
**Pull request by webdevagent**
### Froger OOP task

[Demo](https://webdevagent.github.io/frontend-nanodegree-arcade-game/) |
[Code base](https://github.com/webdevagent/frontend-2019-homeworks/blob/froger-oop-task/submissions/webdevagent/froger-oop/js/app.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by webdevagent** [comment]
You can check it on a mobile phone. I made controls for mobile devices.

File: submissions/webdevagent/froger-oop/js/app.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+};
+
+//Defining gameActor objects
+const allEnemies = [new Enemy(50), new Enemy(130), new Enemy(220)];
+const player = new Hero(300);
+const pointStar = new Star();
+
+//Adding EventListeners to for player navigation
+//navigation for keybord
+document.addEventListener('keyup', function(e) {
```
Comment (IgorKurkov) [inline submissions/webdevagent/froger-oop/js/app.js]: good, good, gooood :)

File: submissions/webdevagent/froger-oop/js/engine.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+         * requires 3 parameters: the image to draw, the x coordinate
+         * to start drawing and the y coordinate to start drawing.
+         * We're using our Resources helpers to refer to our images
+         * so that we get the benefits of caching these images, since
+         * we're using them over and over.
+         */
+        ctx.drawImage(Resources.get(rowImages[row]), col * 101, row * 83);
+
+        ctx.font = "30px Arial";
+        ctx.fillText(`Points: ${point}`, 10, 30);
```
Comment (IgorKurkov) [inline submissions/webdevagent/froger-oop/js/engine.js]: good :+1: but in general Arial not so good font for frontend

**Comment by webdevagent** [comment]
Thank's @IgorKurkov !) Happy New YEAR!) 
I have been fixed mistakes that you founded in my code, Changed the position of scripts, assigned an ununderstandable value to variables, change canvas font to more useful font. Also, I defined a function that helped me do not repeat myself in lines of code.

**Pull request merged**